apps/pie-docs/src/foundations/spacing/overview.md

```diff
@@ -10,47 +10,47 @@ There are several types of spacing we can use to place elements within layouts a
 
 ### Padding
 
-Padding is the distance between elements within a component, which it can be applied vertically and horizontally.
+Padding is the distance between elements within a component, and can be applied vertically any horizontally.
```

Comment:
Is this definition accurate? I'd say the distance between elements is margin, and padding is the distance between the edge of a component and its content.
```suggestion
Padding is the distance between elements within a component, and can be applied both vertically and horizontally.
```

Reply:
Yep, what you suggested is more accurate. Would you mind putting it into words and sharing so it can be updated? Thanks!

Reply:
👍